openedx/core/djangoapps/organization_api/api/views.py

```diff
@@ -0,0 +1,39 @@
+"""
+Organizations API views.
+"""
+from rest_framework import permissions
+from rest_framework.authentication import SessionAuthentication
+from rest_framework.response import Response
+from rest_framework.status import HTTP_404_NOT_FOUND
+from rest_framework.views import APIView
+from rest_framework_jwt.authentication import JSONWebTokenAuthentication
+from rest_framework_oauth.authentication import OAuth2Authentication
+
+from util.organizations_helpers import get_organization_by_short_name
+
+
+class OrganizationsView(APIView):
```

Comment:
@clintonb I understand your concern about fully taking advantage of DRF. The question is that should we:
* import the organizations model here and create DRF serializers?
* add DRF serializer in the `edx-organizations` and use that serializer here?
* remove all this code and expose DRF API from `edx-organizations` itself?

Fort the second and third approach we have to add drf support for the `edx-organizations` and we will handle this as a separate story.
fyi: @mattdrayer @ahsan-ul-haq